reference/render/aanvragen.html

```diff
@@ -3,14 +3,18 @@
     <img src="../../images/omgevingsloket.png" alt="Omgevingsloket" class="logo" />
     <div class="dso-tools-bar">
       <div class="dso-login">
-        <a href="" class="btn btn-link"><span class="icon far fa-user" aria-hidden="true"></span><span>Inloggen</span></a>
+        <a href="" class="btn btn-link"><svg class="di di-user">
+            <use href="../../dso-icons.svg#user" />
+          </svg><span>Inloggen</span></a>
       </div>
     </div>
   </div>
   <nav class="dso-navbar">
     <div class="dso-navbar-header">
       <button type="button" class="dso-navbar-toggle">
-        <span class="fas fa-bars" aria-hidden="true"></span> <span class="sr-only">Ga naar menu</span>
+        <svg class="di di-bars">
+          <use href="../../dso-icons.svg#bars" />
+        </svg> <span class="sr-only">Ga naar menu</span>
```

Comment:
Hier een spatie, maar voor de casus met `<span>Inloggen</span>` niet?

Reply:
Ja.

Om de vorige situatie te blijven ondersteunen is er een uitzondering op icons in buttons/anchors  ZONDER `.btn`. De CSS houdt alleen rekening met een ruimte van 8px tussen icon en label als het in een `.btn. zit.

Als een button/anchor zonder `.btn` wordt gemaakt, werd er voorheen zelf voor ruimte van een spatie gezorgd.

Maar een spatie naast een margin van 8px zorgt ervoor dat er teveel ruimte tussen het label en het icoon komt

Reply:
Ok. Als het maar doordacht is.